### PR TITLE
Added support for authentication and database number selection.

### PR DESCRIPTION
**Pull request by faelenor**

### CODE REVIEW

File: src/main/java/com/chiwanpark/flume/plugins/RedisSubscribeDrivenSource.java [hunk 1]
```diff
@@ -68,8 +72,15 @@ public synchronized void start() {
 
     // TODO: consider using Connection Pool.
     jedis = new Jedis(redisHost, redisPort, redisTimeout);
+    if (redisPassword != "") {
```
Comment (chiwanpark) [inline src/main/java/com/chiwanpark/flume/plugins/RedisSubscribeDrivenSource.java]: We use 2 spaces to indentation, please change this block to use 2 spaces indentation.

**Comment by chiwanpark** [comment]
Thanks for your great work! I had some just minor remarks.

**Comment by faelenor** [comment]
I fixed the indentation. I realized that I only modified the source and not the sink (I only need the source). The same code could probably be copied in the sink if the feature is eventually needed by someone.

**Comment by chiwanpark** [comment]
Great! Thanks for your work!

**Comment by faelenor** [comment]
Oh, I just realized that the pub/sub feature of Redis is not tied to a database! It means that the database config I added is useless and should be removed. The support for authentication is still useful though.

**Comment by chiwanpark** [comment]
@faelenor Okay. I'll remove database configuration.

**Pull request merged**